### ISSUE TITLE
--seperate results in GoError: The moduleSpecifier "./libs/shim/urijs.js" couldn't be found on local disk.

### ISSUE THREAD

**Issue by thim81** [k6io/postman-to-k6#78]
When using the `--separator` like `postman-to-k6 postman.json -s -e environment.json -o k6-script.js`

A range of folders and files are generated and grouped in the k6-script.js

```
// Auto-generated by the Load Impact converter

import "./libs/shim/core.js";
import { group } from "k6";
import "./requests/DomainClusters/Get-all-domain-cluster-configurations.js";

export let options = { maxRedirects: 4 };

const Request = Symbol.for("request");
postman[Symbol.for("initial")]({
  options,
  collection: {
    undefined: "https://example.com/api"
  },
  environment: {
    baseUrl: "https://example.com/portal"
  }
});

export default function() {
  group("DomainClusters", function() {
    postman[Request]("Get all domain cluster configurations");
  });
}
```

In the `Get-all-domain-cluster-configurations.js` file the following code is present

```
import "./libs/shim/urijs.js";

postman[Symbol.for("define")]({
  name: "Get all domain cluster configurations",
  id: "cbe7c568-113f-4352-a0cc-bd8326ca9e7d",
  method: "GET",
  address: "{{baseUrl}}/portal/v1/domainclusters"
});
```

All files are present in the libs/shim folder.

When I run the ` k6 run --vus 100 --duration 1m k6-script.js`, I get the following error.

`ERRO[0006] GoError: The moduleSpecifier "./libs/shim/urijs.js" couldn't be found on local disk. Make sure that you've specified the right path to the file. If you're running k6 using the Docker image make sure you have mounted the local directory (-v /local/path/:/inside/docker/path) containing your script and modules so that they're accessible by k6 from inside of the container, see https://k6.io/docs/using-k6/modules#using-local-modules-with-docker. at reflect.methodValueCall (native) `

Info: I'm not using the Docker version, but installed k6 via Brew and using the 1.1.0 release of postman-to-k6 (which is the latest release according to NPM)

**Comment by mikpan** [comment]
+1, same issue here

**Comment by zakievvv** [comment]
+1, same issue here happens for me when I attempt to use any way of running it with docker for script execution, and clearly supply it with folder that contains these libs folders which are being imported (performance-tests contains libs). 

`docker-compose run -v /Users/myUser/documents/github/mysysco-shopping-mobile-bff/performance-tests:/scripts k6 run /scripts/loadTestDebugV2.js`

It does not look like it is supported in the first place based on readme referring only 
`k6 run script.js` command and not for example
`docker run loadimpact/k6 run - < convertedScript.js`

**Comment by thim81** [comment]
@mikpan @zakievvv Did you find a solution for the libs error? Just curious

**Comment by Eomm** [comment]
I have replaced this import:
```diff
-import './libs/shim/urijs.js'
+import '../../libs/shim/urijs.js'
```

in the `request/` folder only as a workaround

**Comment by thim81** [comment]
Created a PR for this issue: https://github.com/k6io/postman-to-k6/pull/84

Thanks @Eomm for the simple solution.

**Comment by thim81** [comment]
Informational update:
The PR was pushed against the latest master (1.3.0) which is not yet release on NPM. On NPM https://www.npmjs.com/package/postman-to-k6 release 1.10 is still the latest.

Until v1.3.0 and potentially the PR is merged, I provided a separate 1.1.0 branch

```
  "dependencies": {
    "postman-to-k6": "git://github.com/thim81/postman-to-k6#1.1.0",

```

I'm not in favour of using PR directly but I'm just sharing in case people want a temporary work-a-round.
Once there is a new release on NPM (with the PR) I'll remove the fork.
